tests/functional/modules/test_zos_lineinfile_func.py

```diff
@@ -18,6 +18,10 @@
 import pytest
 import inspect
 
+from ibm_zos_core.tests.helpers.dataset import (
+    get_tmp_ds_name,
+    get_random_hlq)
+
 __metaclass__ = type
 
 DEFAULT_DATA_SET_NAME = "USER.PRIVATE.TESTDS"
```

Comment:
This seems to be not in use anymore, all the functions override this, maybe remove it and lower case DEFAULT_DATA_SET_NAME to default_data_set_name , given lower case is usually local and upper case global.